### ISSUE TITLE
7.0.0 fails to install on linux

### ISSUE THREAD

**Issue by fierysolid** [imagemin/imagemin-gifsicle#40]
The following error occurs when trying to install the latest dep of gifsicle on Linux. 6.0.1 works fine

```
error ./node_modules/gifsicle: Command failed.
Exit code: 1
Command: node lib/install.js
Arguments:
Directory: ./node_modules/gifsicle
Output:
⚠ Response code 404 (Not Found)
  ⚠ gifsicle pre-build test failed
  ℹ compiling from source
  ✖ Error: Command failed: /bin/sh -c autoreconf -ivf
/bin/sh: 1: autoreconf: not found
```

**Comment by danez** [comment]
I have the same.

**Comment by nixi-app** [comment]
Can confirm, downgrading to `6.0.1` _works_.

**Comment by wouterds** [comment]
Can confirm, downgrading to `6.0.1` _"works"_.

**Comment by melissaclausse** [comment]
Having the same issue. Bundled within `gulp-imagemin`.

**Comment by nodeg** [comment]
`autoreconf` is not found as the last line above suggests. `autoreconf` is part of `autoconf`, try installing `autoconf` on your distribution

**Comment by melissaclausse** [comment]
The issue is we shouldn't have to manually install these packages. If they are necessary for this package to work, they should be dependencies.

**Comment by nodeg** [comment]
@melissaclausse: The packages I mentioned earlier are not npm packages, but packages that have to be installed using the operating system's package manager.

**Comment by melissaclausse** [comment]
Gotcha - thanks for clarifying. Still, this is fully incompatible with production builds that are not happening on my local machine. Is there anything that could fix this that would be set up as a dependency instead?

**Comment by nodeg** [comment]
`imagemin-gifsicle` depends on `gifsicle-bin` which removed i.a. the [GNU/Linux binary files](https://github.com/imagemin/gifsicle-bin/commit/58099ed974b2b237d05b5a75539010e5126620e0) of `gifsicle` due to [security vulnerabilites](https://github.com/imagemin/gifsicle-bin/issues/87) in version 5.0.0. That is why the binary has to be build from source and why you need `dh-autoreconf`. There is already an upstream issue opened for that: [#113](https://github.com/imagemin/gifsicle-bin/issues/113)

**Comment by janwidmer** [comment]
`gifsicle-bin` has been updated via https://github.com/imagemin/gifsicle-bin/pull/118 and released in version `5.2.0` which should include the Linux bin files again. Therefore please adjust the dependency version to `^5.2.0` in the package json. As we are using `imagemin-gifsicle`, I can only close the issue [#113](https://github.com/imagemin/gifsicle-bin/issues/113) after getting a new release of `imagemin-gifsicle`
